enhancements/two-node-fencing/tnf.md

```diff
@@ -687,76 +781,94 @@ Additionally, it would be good to have workload-specific testing once those are
 
-- Describe which OCP teams are likely to be called upon in case of escalation with one of the failure modes
-  and add them as reviewers to this enhancement.
+- Describe which OCP teams are likely to be called upon in case of escalation with one of the failure modes and add them as reviewers to this enhancement.
 
-  In case of escalation, the most likely team affected outside of Edge Enablement (or whoever owns the proposed topology) is the Control Plane team because etcd is the primary component that two-node OpenShift needs to manage properly to protect against data corruption/loss.
+  In case of escalation, the most likely team affected outside of Edge Enablement (or whoever owns the proposed topology) is the Control Plane team because etcd is the primary component that two-node
+  OpenShift needs to manage properly to protect against data corruption/loss.
 
 ## Support Procedures
 
 - Failure logs for pacemaker will be available in the system journal. The installer should report these in the case that a cluster cannot successfully initialize pacemaker.
-- A BMC connection failure detected after the cluster is installed can be remediated as long as the cluster is healthy. A new MachineConfig can be applied to update the secrets file. If the cluster
-  is down, this file would need to be updated manually.
+- A BMC connection failure detected after the cluster is installed can be remediated as long as the cluster is healthy. A new MachineConfig can be applied to update the secrets file. If the cluster is
+  down, this file would need to be updated manually.
 - In the case of a failed two-node cluster, there is no supported way of migrating to a different topology. The most practical option would be to deploy a fresh environment.
 
 ## Alternatives
 
-* MicroShift was considered as an alternative but it was ruled out because it does not support multi-node and has a very different experience than OpenShift which does not match the TNF initiative which is on getting the OpenShift experience on two nodes
+#### Active-Passive Single-Node OpenShift
+One of the first options that comes to mind when thinking about how to run OpenShift with two control plane nodes at scale without the risk of split-brain is deploying 2 Single-Node OpenShift
+instances and setting them up as an active node with a passive backup.
+
+This option requires the user to run external software like keepalived in front of the instances to ensure that only one instance is active and receiving traffic at a time.
+
+The advantage of this this kind of set up is that failures of each instance is reliably deterministic. A failure of your active instance just passes traffic over to your passive instance, and a
+failure of your passive instance doesn't interrupt the active instance at all. Since both instances cannot be active at the same time, there is no risk of a split-brain situation. Additionally, each
```

Comment:
s/active/the active instance/ to make it clear that we don't mean it in the "running" sense.